### PR TITLE
BaSP-MR-37: Class tests get&post

### PR DESCRIPTION
**Pull request by Flor-Farruggia**
Create the test for the class entity for the Get, Get by Id and the post

### CODE REVIEW

**Review by octavi01 — APPROVED** [review]

File: src/models/Class.js [hunk 1]
```diff
@@ -14,9 +14,9 @@ const classSchema = new Schema({
     required: true,
   },
   trainer: {
-    type: Schema.Types.ObjectId,
+    type: [Schema.Types.ObjectId],
+    ref: 'trainer',
```
Comment (JuanMornik) [inline src/models/Class.js:18]: Replace this with 'Trainer' to avoid conflicts in a future

File: src/tests/class.test.js [hunk 2]
```diff
@@ -1,5 +1,60 @@
...
+import app from '../app';
+import Class from '../models/Class';
+import classSeed from '../seeds/class';
+
+beforeAll(async () => {
+  await Class.collection.insertMany(classSeed);
+});
+
+describe('GET /api/class', () => {
+  test('This should return status 200', async () => {
```
Comment (JuanMornik) [inline src/tests/class.test.js:11]: Status 500 test missing

File: src/tests/class.test.js [hunk 3]
```diff
@@ -1,5 +1,60 @@
...
+    const response = await request(app).get('/api/class').send();
+    expect(response.status).toBe(200);
+    expect(response.error).toBeFalsy();
+  });
+});
+
+describe('GET BY ID /api/class/:id', () => {
```
Comment (JuanMornik) [inline src/tests/class.test.js:18]: Status 500 test missing

File: src/tests/class.test.js [hunk 4]
```diff
@@ -1,5 +1,60 @@
...
+  });
+  test('Check invalid Route', async () => {
+    const id = '646596b54fcb63fdd73b28a6';
+    const response = await request(app).get(`/api/classes/${id}`).send();
+    expect(response.status).toBe(404);
+    expect(response.error).toBeTruthy();
+  });
+});
+
+describe('POST /api/class', () => {
```
Comment (JuanMornik) [inline src/tests/class.test.js:39]: Status 500 test missing

**Review by JuanMornik — CHANGES_REQUESTED** [review]

File: src/tests/class.test.js (hunk 3 above)
Comment (JuanMornik) [reply src/tests/class.test.js:25]: valid ID test

**Review by JuanMornik — CHANGES_REQUESTED** [review]

**Review by JuanMornik — APPROVED** [review]

**Pull request merged**